spancontext.go

```diff
@@ -60,14 +62,33 @@ type DatabaseSpanContext struct {
 	User string
 }
 
+// The span service target fields replace the `span.destination.service.*`
```

Comment:
```suggestion
// ServiceSpanContext holds ...
//
// The span service target fields replace the `span.destination.service.*`
```

Reply:
Ah :facepalm: , thanks for spotting this!